### PR TITLE
chore: remove `README.md` file

### PR DESCRIPTION
**Pull request by Khayal-Dev**

### CODE REVIEW

**Review by darkyeg — CHANGES_REQUESTED** [review]
Make a new one.

File: apps/rest/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+<h3 align="center">Rest API</h3>
```
Comment (darkyeg) [inline apps/rest/README.md]: make it `h1`

**Review by darkyeg — CHANGES_REQUESTED** [review]
Add some details

File: apps/rest/README.md (hunk 1 above)
Comment (Khayal-Dev) [reply apps/rest/README.md]: Okay I will change it from `h3` to `h1`

**Review by darkyeg — APPROVED** [review]

**Review by darkyeg — APPROVED** [review]
Great!

**Pull request merged**